sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_condition.py

```diff
@@ -20,32 +20,42 @@ def condition(
 ) -> ConditionNode:
     """Create a condition node to provide runtime condition graph experience.
 
-    Below is an example of using expression result to control which step is executed.
-    If pipeline parameter 'int_param1' > 'int_param2', then 'true_step' will be executed,
-    else, the 'false_step' will be executed.
+    Below is an example of using an expression result to control which step is executed.
+    If the pipeline parameter 'int_param1' is greater than 'int_param2', then 'true_step' will be executed,
+    otherwise, the 'false_step' will be executed.
 
     .. code-block:: python
 
-    @dsl.pipeline
-    def pipeline_func(int_param1: int, int_param2: int):
-        true_step = component_func()
-        false_step = another_component_func()
-        dsl.condition(
-            int_param1 > int_param2,
-            true_block=true_step,
-            false_block=false_step
-        )
+        @dsl.pipeline
+        def pipeline_func(int_param1: int, int_param2: int):
+            true_step = component_func()
+            false_step = another_component_func()
+            dsl.condition(
+                int_param1 > int_param2,
+                true_block=true_step,
+                false_block=false_step
+            )
 
     :param condition: The condition of the execution flow.
-        The value could be a boolean type control output, a node with exactly one boolean type output
+        The value could be a boolean type control output, a node with exactly one boolean type output,
         or a pipeline expression.
-    :type condition: Union[str, bool, InputOutputBase, BaseNode, PipelineExpression]
-    :keyword true_block: Block to be executed if condition resolved result is true.
-    :type true_block: BaseNode
-    :keyword false_block: Block to be executed if condition resolved result is false.
-    :type false_block:  BaseNode
+    :type condition: Union[
+        str,
+        bool,
+        ~azure.ai.ml.entities._job.pipeline._io.InputOutputBase,
+        ~azure.ai.ml.entities._builders.BaseNode,
+        ~azure.ai.ml.entities._job.pipeline._pipeline_expression.PipelineExpression]
+    :param true_block: The block to be executed if the condition resolves to True. Defaults to None.
```

Comment:
After offline discussion, we have decided to delete all` Defaults to None` because a default value will be assigned internally to the code logic.
If 'None' is written here, it will actually cause confusion for the user.